test/README.md

```diff
@@ -0,0 +1,26 @@
+# SLB
+
+> C++ Standard Library Backport
+
+[![travis][badge.travis]][travis]
+[![appveyor][badge.appveyor]][appveyor]
+[![license][badge.license]][license]
+
+[badge.travis]: https://travis-ci.org/mpark/slb.svg?branch=master
+[badge.appveyor]: https://ci.appveyor.com/api/projects/status/github/mpark/slb?branch=master&svg=true
+[badge.license]: https://img.shields.io/badge/license-boost-blue.svg
+
+## Test
+
+This directory contains the tests for __SLB__.
+
+## Build / Run
+
+Execute the following commands from the top-level directory:
+
+```bash
+mkdir build && cd build
+CXXFLAGS="-std=c++11" cmake ..
```

Comment:
Setting appropriate `CXXFLAGS` is not a necessary step of testing, nor is adding `-std=C++11` required (for modern gcc/clang versions) or correct (for msvc). It is no different than configuring the environment such that a C++ compiler is available.

This could be a note instead, stating that _at least_ a C++11 conforming compiler is needed, and that the `CXX/CXXFLAGS` environment variables _may_ be used to that effect.

Reply:
For now I just removed the `CXXFLAGS`. As you point out, the steps to build / run the tests
with MSVC for example are different with `--config Debug` being specified at build time, etc.

I'm not even sure whether that's a MSVC specific thing, or a Windows specific thing.
It seems to me like it's an MSVC thing since using `"MinGW Makefiles"` uses the same
steps as outlined here.

Anyway, I think these steps are probably sufficient for now.

Reply:
It is a CMake specific thing for multi-configuration generators, which the Visual Studio ones are (and I believe the only others are the XCode ones).